src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -111,4 +112,12 @@ export class SortableTableComponent implements OnInit {
     this.columnToSortBy = this.columns[indexOfColumnToSort].header;
     this.sortRows();
   }
+
+  getStyle(cellData: SortableTableCellData): string {
```

Comment:
is it possible to return `string | undefined` here? so that u can get rid of `return ''`